test/selenium/ContributorSpec.scala

```diff
@@ -62,21 +62,7 @@ class ContributorSpec extends FeatureSpec with Browser with GivenWhenThen with B
       When("they press the Stripe payment button")
       MonthlyContribution.selectStripePayment()
 
-      Then("the Stripe Checkout iFrame should display")
-      assert(MonthlyContribution.stripeCheckoutHasLoaded)
```

Comment:
There's a whole bunch of test code related to Stripe Checkout which can also go if people are happy with this approach.

Reply:
If it is code that we are not currently using, I would delete it in this PR as well.

Reply:
👍 https://github.com/guardian/support-frontend/pull/335/commits/7e92c792267a75b9d205083a1979d78a24c2bdb8 :hocho: